pages.de/common/git-log.md

```diff
@@ -0,0 +1,36 @@
+# git log
+
+> Zeigt die Commit-Historie an.
```

Comment:
The verbs should be in `present indicative`, not `present continuous`. I don't speak German but I should check.

Reply:
@LukBukkit, based on your PR #4793. Should we replace `Zeigt` by `Zeige` here?

Reply:
I'm sorry for my late answer. 
Yeah, I would prefer the usage of `Zeige` as it is the [imperative of `Zeigen`](https://www.duden.de/rechtschreibung/zeigen#grammatik) and so it does sound more targeted to the user. A lot of other German translations are also using the imperative:
* [`7z`](https://github.com/tldr-pages/tldr/blob/master/pages.de/common/7z.md)
* [`bash`](https://github.com/tldr-pages/tldr/blob/master/pages.de/common/bash.md)
* [`bat`](https://github.com/tldr-pages/tldr/blob/master/pages.de/common/bat.md)

But not every translated pages does, for example the page [`git`](https://github.com/tldr-pages/tldr/blob/master/pages.de/common/git.md) also uses `Zeigt`.

Reply:
@CraxyTM Could you please apply this change?

Reply:
Hrm. If other translations are using the imperative, then it's probably fine to use the imperative here too? I mean I don't speak German and I have no idea what the imperative, present simple, or indicative are, so I'm not sure can comment here.

Reply:
I replaced "Zeigt" with "Zeige" as requested by @navarroaxel